maitake/src/wait/cell.rs

```diff
@@ -237,8 +303,37 @@ impl fmt::Debug for State {
     }
 }
 
+#[cfg(all(feature = "alloc", not(loom), test))]
+mod test {
```

Comment:
this is a _very_ stupid nit, and i'm sorry in advance, but...every other module in this crate calls the test module `tests` rather than `test`, and it's kind of nice to be consistent with that when running with `cargo test wait::cell::tests` or similar...